src/components/NatureOfBusiness/NatureOfBusiness.vue

```diff
@@ -33,17 +28,15 @@
             @keydown.enter="onSearchClicked()"
           >
             <template #append>
-              <v-btn
-                id="btn-search"
-                color="primary"
-                depressed
-                :loading="state === States.SEARCHING"
-                @click="onSearchClicked()"
-              >
-                <v-icon>mdi-magnify</v-icon>
-              </v-btn>
+              <v-btn depressed id="btn-search" color="primary" :loading="state === States.SEARCHING"
+                @click="onSearchClicked()"><v-icon>mdi-magnify</v-icon></v-btn>
             </template>
           </v-text-field>
+          <template v-if="state === States.INITIAL && haveNaics">
+            <v-btn large outlined color="primary" class="float-right mt-8" @click="onCancelClicked()">
+              <span>Cancel</span>
+            </v-btn>
+          </template>
```

Comment:
Thank you very much! 
The `mt-8` here isn't required for the EDIT UI. 
If this isn't something that will be utilized in the Create UI could we remove it?

Reply:
If a value was previously stored and the user clicks Change, then we want to show Cancel.  Are you suggesting less whitespace, mt-6 maybe?

![image](https://user-images.githubusercontent.com/10002889/157949518-ec440de5-36b2-4dbb-a6c1-4f6154920d7d.png)

Reply:
Definitely. 
Design calls for 25px from bottom of the input field line to the top of the button. 
From my mock up, this was achieved without any additional top margin.

Reply:
You are correct. No mt matches the comps:

![image](https://user-images.githubusercontent.com/10002889/157950345-9b77af30-a7ac-4da1-bc88-41dda2adca64.png)

Reply:
Cool, ty!